x-pack/plugin/transform/src/main/java/org/elasticsearch/xpack/transform/rest/action/RestCatTransformAction.java

```diff
@@ -62,7 +62,7 @@ protected RestChannelConsumer doCatRequest(RestRequest restRequest, NodeClient c
         GetTransformAction.Request request = new GetTransformAction.Request(id);
         request.setAllowNoResources(restRequest.paramAsBoolean(ALLOW_NO_MATCH.getPreferredName(), true));
 
-        GetTransformStatsAction.Request statsRequest = new GetTransformStatsAction.Request(id, null);
+        GetTransformStatsAction.Request statsRequest = new GetTransformStatsAction.Request(id, null, false);
```

Comment:
I am intentionally leaving this to false?

I'm not sure when we should introduce `basic` as a query parameter for the `_cat` API: https://www.elastic.co/guide/en/elasticsearch/reference/current/cat-transforms.html

I figured I could follow up with it? Alongside documentation for this change

Reply:
I don't think you should add a `basic` option to the `_cat` API.

The `_cat` APIs let the user choose the specific columns to show: https://www.elastic.co/guide/en/elasticsearch/reference/current/cat.html#headers

The implementation of `_cat` can then use `basic` on the underlying stats call if that will deliver enough information to return the values for the requested columns. We might need to ask an expert from the ES team how to do this, as it looks like the implementation adds all the possible columns and leaves it up to the framework to decide which ones to display.

But, like you say, `_cat` can be a follow-up PR.